pydicer/convert/data.py

```diff
@@ -0,0 +1,45 @@
+import pydicom
+import SimpleITK as sitk
+from platipy.dicom.io.crawl import safe_sort_dicom_image_list
+from pathlib import Path
+import hashlib
+
+
+class ConvertData:
+    """
+    Class that facilitates the conversion of the data into its intended final type
+
+    Args:
+        - preprocess_dic: the dictionary that contains the preprocessed data information
+    """
+
+    def __init__(self, preprocess_dic, output_directory="."):
+        self.preprocess_dic = preprocess_dic
+        self.output_directory = Path(output_directory)
+
+    def convert(self):
+        """
+        Function to convert the data into its intended form (eg. images into Nifti)
+
+        return: void
+        """
+        m = hashlib.sha256()
+
+        for key, value in self.preprocess_dic.items():
```

Comment:
I'm just wondering about the variable names: m, key, value... In this small context here its obvious that they do becuase its easy to read. But Im thinking as we extend this code it might be nice to have more description names here?

Perhaps: hash_sha, series_uid and file_dic?